diff --git a/dist.linux/share/bash-completion/cellframe-node-cli b/dist.linux/share/bash-completion/cellframe-node-cli
new file mode 100755
index 0000000..86c215e
--- /dev/null
+++ b/dist.linux/share/bash-completion/cellframe-node-cli
@@ -0,0 +1,23 @@
+_cellframe-node-cli() {
+	local cur prev opts
+
+	COMPREPLY=()
+	cur="${COMP_WORDS[COMP_CWORD]}"
+
+	prev="${COMP_WORDS[COMP_CWORD-1]}"
+
+	case "${prev}" in
+		"cellframe-node-cli")
+			opts="dag dag_poa block block_poa net_srv srv_xchange srv_stake stake_lock srv_datum net vpn_client global_db mempool node ping traceroute tracepath version help wallet token_update token_decl token_decl_sign token_emit mempool_list mempool_proc mempool_delete mempool_add_ca chain_ca_pub chain_ca_copy tx_create tx_create_json tx_cond_create tx_verify tx_history ledger token print_log stats exit gdb_export gdb_import"
+			COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+			return 0
+		    ;;
+ 		*)
+		;;
+	esac
+
+	# No matches? Return nothing.
+	return 0
+}
+
+complete -F _cellframe-node-cli cellframe-node-cli
Submodule prod_build contains modified content
diff --git a/prod_build/linux/debian/essentials/postinst b/prod_build/linux/debian/essentials/postinst
index be06e88..ffacb91 100755
--- a/prod_build/linux/debian/essentials/postinst
+++ b/prod_build/linux/debian/essentials/postinst
@@ -211,4 +211,10 @@ ln -sf $DAP_PREFIX/share/logrotate/$DAP_APP_NAME /etc/logrotate.d/$DAP_APP_NAME
 chmod 644 /etc/logrotate.d/$DAP_APP_NAME || true
 systemctl reenable logrotate.timer || true 
 
-echo "[!] Done"
\ No newline at end of file
+
+#Adds basic support for bash-completion with cellframe-node-cli
+ls -sf $DAP_PREFIX/share/bash-completion/cellframe-node-cli /usr/share/bash-completion/completions/cellframe-node-cli || true
+source ./usr/share/bash-completion/completions/cellframe-node-cli || true
+
+
+echo "[!] Done"
