diff --git a/CMakeLists.txt b/CMakeLists.txt
index dba0a12..1f6ede8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -115,7 +115,7 @@ if(UNIX)
     SET( CPACK_DEBIAN_PACKAGE_SECTION "extra")
     SET( CPACK_DEBIAN_PACKAGE_DEPENDS "debconf (>=1.5), dpkg (>=1.17), bash (>=4), debconf-utils, dconf-cli, less, pv, psmisc, logrotate, irqbalance, xz-utils")
      if (CMAKE_TARGET_ARCH MATCHES arm)
-        SET( CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libmagic1, libpython3.9" )
+        SET( CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libmagic1, libpython3-dev" )
     endif()
     SET( CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
     SET( CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA
