diff --git a/linux/debian/essentials/postinst b/linux/debian/essentials/postinst
index b2c2a51..13909ce 100755
--- a/linux/debian/essentials/postinst
+++ b/linux/debian/essentials/postinst
@@ -9,6 +9,7 @@ DAP_CFG_TPL="$DAP_PREFIX/share/configs/$DAP_APP_NAME.cfg.tpl"
 
 # Store write config to new if present smth
 DAP_CFG="$DAP_PREFIX/etc/$DAP_APP_NAME.cfg"
+DAP_CFG_BACKUP="$DAP_PREFIX/etc/${DAP_APP_NAME}_$(date +"%Y%m%d_%H%M%S").cfg"
 
 #fixed link with python library
 BUILD_PYTHON_VERSION=/usr/lib/x86_64-linux-gnu/libpython3.5m.so.1.0
@@ -31,14 +32,12 @@ fi
 
 # Init node config
 if [ -e "$DAP_CFG" ]; then
-    DAP_CFG="$DAP_PREFIX/etc/$DAP_APP_NAME.cfg.dpkg-new"
-else
-    DAP_CFG="$DAP_PREFIX/etc/$DAP_APP_NAME.cfg"
+    mv $DAP_CFG $DAP_CFG_BACKUP
 fi
 
 cat $DAP_CFG_TPL > $DAP_CFG || true
 
-sed -i "s/{PREFIX}/\/opt\/$DAP_APP_NAME/g" $DAP_CFG  || true
+sed -i "s|{PREFIX}|/opt/$DAP_APP_NAME|g" $DAP_CFG  || true
 
 db_get cellframe-node/debug_mode || true
 sed -i "s/{DEBUG_MODE}/$RET/g" $DAP_CFG  || true
